main.go

```diff
@@ -238,6 +270,51 @@ func getDeviceFromName(devices []Device, name string) (*Device, error) {
 	return &matching_udids[0], nil
 }
 
+func mosyleLogin() error {
+	data, err := json.Marshal(map[string]string{
+		"email":    os.Getenv("MOSYLE_EMAIL"),
+		"password": os.Getenv("MOSYLE_PASSWORD"),
+	})
+	if err != nil {
+		return err
+	}
+
+	req, err := http.NewRequest(http.MethodPost, "https://businessapi.mosyle.com/v1/login", bytes.NewBuffer(data))
+	if err != nil {
+		return err
+	}
+
+	req.Header.Set("Content-Type", "application/json")
+	// !!!: We don't use `Set` here because Mosyle is sensitive to the case of the
+	// `accesstoken` header -- http will canonicalize `accesstoken` to
+	// `Accesstoken`, and Mosyle won't accept that.
+	// https://stackoverflow.com/a/26352765
+	req.Header["accessToken"] = []string{os.Getenv("MOSYLE_ACCESS_TOKEN")}
+
+	client := http.Client{
+		Timeout: 30 * time.Second,
+	}
+
+	res, err := client.Do(req)
+	if err != nil {
+		return err
+	}
+
+	if res.StatusCode == 200 {
+		if len(res.Header["Authorization"]) == 1 {
+			jwt = res.Header["Authorization"][0]
```

Comment:
Does the JWT not need `Bearer ...`, or is that returned as part of the response?

Reply:
it is part of the response